From 5ef07c855e2188569f3ed9e0a0476827e87ade4e Mon Sep 17 00:00:00 2001
From: "Teoh, Ji Sheng" <ji.sheng.teoh@intel.com>
Date: Mon, 7 Mar 2022 21:17:35 +0000
Subject: [PATCH] arm: agilex: add board configuration

This patch adds the following two configuration:
board-3: PR design
board-4: Combined GHRD design

Signed-off-by: Teoh, Ji Sheng <ji.sheng.teoh@intel.com>
---
 arch/arm/dts/socfpga_agilex_socdk-u-boot.dtsi | 24 +++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/arch/arm/dts/socfpga_agilex_socdk-u-boot.dtsi b/arch/arm/dts/socfpga_agilex_socdk-u-boot.dtsi
index 6efe2f0952..98d1d8748c 100644
--- a/arch/arm/dts/socfpga_agilex_socdk-u-boot.dtsi
+++ b/arch/arm/dts/socfpga_agilex_socdk-u-boot.dtsi
@@ -96,6 +96,30 @@
 			sign-images = "atf", "uboot", "fdt-1";
 		};
 	};
+
+	board-3 {
+	description = "board_3";
+		firmware = "atf";
+		loadables = "uboot";
+		fdt = "fdt-0";
+		signature {
+			algo = "crc32";
+			key-name-hint = "dev";
+			sign-images = "atf", "uboot", "fdt-0";
+		};
+	};
+
+	board-4 {
+		description = "board_4";
+		firmware = "atf";
+		loadables = "uboot";
+		fdt = "fdt-0";
+		signature {
+			algo = "crc32";
+			key-name-hint = "dev";
+			sign-images = "atf", "uboot", "fdt-0";
+		};
+	};
 };
 
 &binman {
-- 
2.17.1

